### PR TITLE
feat: add recommended Node.js version

### LINKED ISSUE #995: feat: add recommended node.js version for development
There is an engines field in the json package. But to make it easier to install and find versions, I wanted to suggest adding  [Volta](https://volta.sh/)

### PR DESCRIPTION
**Pull request by easymikey**
Add recommended Node.js version for version manager supports `.node_version` and for Volta add package.json field. 

I have prepared a draft for support `.node_version` file for volta

https://github.com/volta-cli/volta/issues/1973

Fixes #995

- [x] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -163,5 +163,8 @@
   },
   "homepage": "https://google.github.io/zx/",
   "author": "Anton Medvedev <anton@medv.io>",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "volta": {
+    "node": "22.12.0"
```
Comment (antongolub) [inline package.json:168]: can we use just `22` here?

File: docs/contribution.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ Before proposing changes, look for similar ones in the project's [issues](https:
   * Bash is essential for running zx scripts. Linux and macOS users usually have it installed by default. Consider using [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install) or [Git Bash](https://git-scm.com/downloads) if you are on Windows.
 * Fork [the repository](https://github.com/google/zx).
 * Create a new branch.
+* Switch to the recommended version of Node.js
+  * If you are using [Node.js version manager](https://github.com/shadowspawn/node-version-usage) supported `.node_version` using it.
+  * If you are using Volta, Node.js version will be set automatically from `package.json`.
+  * In another case manually install recommended version from  `.node_version` file.
```
Comment (antongolub) [inline docs/contribution.md:40]: * Install manually Node.js >= 22
* Delegate the routine to any version manager, that [supports `.node_version` config](https://stackoverflow.com/questions/27425852/what-uses-respects-the-node-version-file)
* Use Volta, the target version will be set automatically from the `package.json`

File: package.json (hunk 1 above)
Comment (easymikey) [reply package.json:168]: No 

I will be uploading an issue soon.


Screenshot 2024-12-20 at 18 01 47

File: docs/contribution.md (hunk 2 above)
Comment (easymikey) [reply docs/contribution.md]: D1

**Review by antongolub — APPROVED** [review]
Accepted. Thanks!

**Pull request merged**